flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/rest/header/operation/CancelOperationHeaders.java

```diff
@@ -52,4 +52,9 @@ public String getTargetRestEndpointURL() {
     public static CancelOperationHeaders getInstance() {
         return INSTANCE;
     }
+
+    @Override
+    public String operationId() {
+        return "CancelOperation";
```

Comment:
I have looked at the override of the `operationId ` method of some existing `XXXHeaders` like `ShutdownHeaders` and found that their first letter is lowercase, should we keep this convention?

Reply:
Done.